README.md

```diff
@@ -6,24 +6,93 @@
+
 The following diagram demonstrates the path a packet takes as it traverses the overlay network:
 
 ![Life of a packet](./packet-01.png)
 
-## Documenation
-- [Running](Documentation/running.md)
+## Getting started
+
+It's really easy to use flannel with Kubernetes. Flannel supports using the Kubernetes API server as its backing datastore which means there's no need to deploy an etcd server for flannel, or even for flannel to access the etcd server that Kubernetes uses. This flannel mode is known as the *kube subnet manager* and is demonstrated below.
+
+### Adding flannel
+
+Flannel can be added to any existing Kubernetres cluster. It's easiest to add flannel before any pods have been started (host networked pods are fine).
+
+A great way to create a new Kubernetes cluster is with [kubeadm][kubeadm] which helpfully points out the need to define *a pod network CIDR* for example by passing in `--pod-network-cidr 10.244.0.0/16` when running `kubeadm init`. This pod network CIDR is the key prerequisite for using flannel with the kube subnet manager. If you're running flannel through some other means, then the pod CIDR might be set automatically for you or you can set it manually by passing [--pod-cidr][pod-cidr] to each of your kubelets.
+
+It's also possible to set it per-node using `kubectl patch`:
+```bash
+kubectl patch node minikube -p '{"spec":{"podCIDR":"10.33.12.0/24"}}'
+```
+
+Once the `podCIDR` is set, add flannel to the cluster using `kubectl apply`:
+```bash
+kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml
+```
+
+If you're using RBAC, also apply kube-flannel-rbac.yaml:
+```bash
+kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel-rbac.yml
+```
+
+Although not required, it's highly recommended that you use CNI to start your pods. This requires your kubelets to be started with `--network-plugin=cni`. (The default if you're using `kubeadm`.)
+
+### What just happened
+
+The manifest creates three things:
+1. A service account for flannel to use.
+2. A ConfigMap containing both a CNI configuration and a flannel configuration. The network in the flannel configuration should match the pod network CIDR. The choice of backend is also made here and defaults to VXLAN.
+3. DaemonSet to deploy the flannel pod on each Node. The pod has two containers 1) the flannel daemon itself, and 2) a container for deploying the CNI configuration to a location that the kubelet can read.
+
+### What can I do now
+
+When you run pods, they will be allocated IP addresses from the pod network CIDR. No matter which node those pods end up on, they will be able to communicate with each other. Try it by running a couple of simple pods and pinging between them.
+
+For example:
+```bash
```

Comment:
This example seems incomplete. Shouldn't it show starting _2_ pods, perhaps the second one interactively, so that you fall into its shell and from there `ping` the first pod?